man/man8/zpool.8

```diff
@@ -272,6 +272,16 @@ However, raidz vdev types are not supported for the intent log.
 For more information, see the
 .Sx Intent Log
 section.
+.It Sy special
+A special device dedicates the device to only participate in the special
+allocation class. The redundancy of this device needs to match the
+redundancy of the other normal devices in the pool. If more than one
+special device is specified, then allocations are load-balanced between
+devices.
```

Comment:
The `dedup` log vdev type should be added here so it's clear that it's a valid type.